src/libs/actions/Report.js

```diff
@@ -425,6 +417,65 @@ function removeOptimisticActions(reportID) {
     });
 }
 
+/**
+ * If an iouReport is open (has an IOU, but is not yet paid) then we maintain a link between it and the associated
```

Comment:
NAB, but the link is not maintained by _this function_

Reply:
Resolved, by simplifying the comment.